src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final String DEFAULT_COLOR = "White";
+    public static final int DEFAULT_RADIUS = 10;
+    private static final int FIGURE_TYPES = 5;
+    private static final double DIMENSION_RANGE = 10.0;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int figureType = random.nextInt(FIGURE_TYPES);
+        String color = colorSupplier.getRandomColor();
+        switch (figureType) {
+            case 0:
+                return createSquare(color);
+            case 1:
+                return createRectangle(color);
+            case 2:
+                return createRightTriangle(color);
+            case 3:
+                return createCircle(color);
+            default:
+                return createIsoscelesTrapezoid(color);
```

Comment:
Don't return 'null' from a method. In the 'switch' construction, you may use the last possible option in the 'default' case, which you have done correctly here.